source/includes/cloudstack/_egress_rules.md

```diff
@@ -0,0 +1,154 @@
+### Egress rules
+
+Manage outbound traffic rules for isolated networks
+
+#### List egress rules
+```shell
+curl -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/compute-on/test_area/egressrules"
+
+# Response example:
+```
+```json
+{
+    "data": [
+       {
+            "id": "f6c7f86f-1e28-42fd-804f-0416a201a778",
+            "networkId": "858d3ab6-04c9-43ef-9e83-07f28df59323",
+            "sourceCidr": "10.2.1.0/25",
+            "destinationCidr": "",
+            "protocol": "ALL",
+            "state": "Active",
+            "allPortsAllowed": false
+        },
+        {
+            "id": "d860f2e7-ad15-4cb0-8477-6aa439245d1f",
+            "networkId": "858d3ab6-04c9-43ef-9e83-07f28df59323",
+            "sourceCidr": "10.2.1.0/24",
+            "destinationCidr": "",
+            "protocol": "ICMP",
+            "icmpType": "3",
+            "icmpCode": "5",
+            "state": "Active",
+            "allPortsAllowed": false
+        },
+        {
+            "id": "f1863d10-e7ec-4f17-8cdd-0e4046643b0b",
+            "networkId": "858d3ab6-04c9-43ef-9e83-07f28df59323",
+            "sourceCidr": "10.2.1.0/24",
+            "destinationCidr": "",
+            "protocol": "UDP",
+            "startPort": "",
+            "endPort": "",
+            "state": "Active",
+            "allPortsAllowed": true
+        }
+    ],
+    "metadata": {
+        "recordCount": 3
+    }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/egressrules</code>
+
+List egress rules.
+
+Attributes | &nbsp;
+---------- | -----
+`id`<br/>*UUID* | The id of the egress rule
+`networkId`<br/>*string* | The id of the network the rule applies to
+`sourceCidr`<br/>*string* | The source cidr
+`destinationCidr`<br/>*string* | The destination cidr
+`protocol`<br/>*string* | The protocol the rule applies to
```

Comment:
You could mention the possible values here.

Reply:
done